packages/pegasus/src/proposals/core-proposal.js

```diff
@@ -69,10 +69,10 @@ export const addPegasusTransferPort = async (
       const { localAddr, actions } = connectionState;
       if (actions) {
         // We're open and ready for business.
-        pegasusConnectionsAdmin.update(localAddr, connectionState);
+        E(pegasusConnectionsAdmin).update(localAddr, connectionState);
```

Comment:
Please use `void` to indicate we don't care about rejected promises, like:

```suggestion
        void E(pegasusConnectionsAdmin).update(localAddr, connectionState);
```

If that's not desired, and rejections should be propagated to stop the iteration observer, please use `return` instead of `void`.

Reply:
Thanks for the review! 

> If that's not desired, and rejections should be propagated to stop the iteration observer, please use return instead of void.

I'm not sure what the correct behavior should be? If we stop the iteration observer, this means that we won't be able to use pegasus for the later connections right? What if there was a one time problem in the `E(...)` call and the later calls will success? Also, do we have backup plan for the connections we could not write to `pegasusConnections` due to some failure `E(pegasusConnectionsAdmin).update()` ?

Reply:
> I'm not sure what the correct behavior should be?

Let's use `void`.

> What if there was a one time problem in the `E(...)` call and the later calls will success? Also, do we have backup plan for the connections we could not write to `pegasusConnections` due to some failure `E(pegasusConnectionsAdmin).update()` ?

`E` is pretty bulletproof, and the worst that it could do is return a rejected promise.  So by using `void`, we guarantee that this code won't fail, but will (correctly) print an unhandledpromiserejection error if the callee fails.